### ISSUE TITLE
[ic-tab-group/ic-horizontal-scroll] A11y violation

### ISSUE THREAD

**Issue by gd2910** [mi6/ic-ui-kit#1850]
## Summary of the bug 
E2E a11y testing on `ic-tabs` has shown a serious violation when using `ic-tabs` with scrollable content. The issue looks to stem from `ic-horizontal-scroll` (see below).

### Violation
![image](https://github.com/mi6/ic-ui-kit/assets/122624946/6f314252-bae5-402b-95d2-76197f0f9617)

### `packages/web-components/src/components/ic-horizontal-scroll.tsx`
![image](https://github.com/mi6/ic-ui-kit/assets/122624946/8fe812e1-0903-441e-b063-9bd77341f124)

## 🖥 📱 Device 
- Type: Cypress a11y testing

## 🧐 Expected behaviour 
All a11y tests should pass. To check this please uncomment the a11y tests in the `ic-tabs` Cypress tests
